src/test/resources/log4j.properties

```diff
@@ -1,9 +1,9 @@
 # Root logger option
-log4j.rootLogger=ERROR, stdout, file
+log4j.rootLogger=DEBUG, stdout, file
```

Comment:
is this still correct? Should keep not to change for love deploys maybe, logs are verbose enough as is there

Reply:
This configuration is used for tests. The one for deploys is on INFO.